activesupport/lib/active_support/core_ext/date/blank.rb

```diff
@@ -11,4 +11,8 @@ class Date # :nodoc:
   def blank?
     false
   end
+
+  def present? # :nodoc:
```

Comment:
In this case `Date` is already nodoc, so we don't need this extra comment.

Technically `Date#blank?` is private API.

Reply:
Right, I got tricked because it has a doc comment. Unclear what the intent was.